unstructured/partition/pdf.py

```diff
@@ -503,11 +503,11 @@ def add_pytesseract_bbox_to_elements(elements, bboxes, width, height):
             max_y = max(max_y, y2)
 
         points = ((min_x, min_y), (min_x, max_y), (max_x, max_y), (max_x, min_y))
-        converted_points = []
+        converted_points = tuple()
         for point in points:
             x, y = point
             new_x, new_y = point_space.convert_coordinates_to_new_system(pixel_space, x, y)
-            converted_points.append((new_x, new_y))
+            converted_points = (*converted_points, (new_x, new_y))
```

Comment:
a bit more on why the comment above:
the step `converted_points = (*converted_points, (new_x, new_y))` would create tuple and iterate (the old) tuple for every step in the loop; this can be costly if the number of `points` is high (basically O(N*2) complexity)

using tuple comprehension reduces that complexity

but maybe we can do better by providing a vectorized conversion api so we can convert points by batch (e.g., using np array as input)?